scripts/lunch.coffee

```diff
@@ -0,0 +1,3 @@
+module.exports = (robot) ->
+    robot.hear /What are we having for lunch?/i, (res) ->
+        res.send("/poll Where should we go for lunch? :gbk: :pizza: :spaghetti: :curry: :sushi: :taco: :burrito: :ramen: :chicken:")
```

Comment:
The regex could have been more flexible 😛  it won't catch 'Where are we going for lunch?'